C/common/include/management_client.h

```diff
@@ -39,9 +39,9 @@ class ManagementClient {
 							   const std::vector<std::string>& children) const;
 		std::vector<AssetTrackingTuple*>&	getAssetTrackingTuples() const;
 		bool addAssetTrackingTuple(const std::string& service, 
-													const std::string& plugin, 
-													const std::string& asset, 
-													const std::string& event);
+												const std::string& plugin, 
+												const std::string& asset, 
+												const std::string& event);
```

Comment:
You can see the code is not well formatted: use tabs

Reply:
I think tabs is an issue, tabs are displayed as 2, 4 or 8 spaces in different editors and viewers and websites like github. Should we expand tabs to fixed number of spaces, say 4?